src/pages/Join/Join.scss

```
@@ -0,0 +1,223 @@
+    width: 100%;
+    display: flex;
+    button {
+      display: flex;
+      height: 56px;
+      padding: 8px 497px 8px 0px;
+      align-items: center;
+      flex-shrink: 0;
+      align-self: stretch;
+      border: none;
+      background-color: white;
+      font-size: 20px;
+      line-height: 28px;
+      cursor: pointer;
+    }
+  }
+
+  .joinForm {
+    display: flex;
+    padding: 0 24px;
+    flex-direction: column;
+    gap: 8px;
+    align-self: stretch;
+
+    .loginInput {
+      width: 100%;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+
+      input {
+        display: flex;
+        height: 56px;
+        border: 1px solid $grey60;
+        border-radius: 6px;
+        padding: 17px 16px;
+        align-items: center;
+        align-self: stretch;
+        font-family: Apple SD Gothic Neo;
+        font-size: 16px;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 22px;
+        margin-bottom: 4px;
+      }
+    }
+
+    .phoneNumber {
+      width: 100%;
+      display: flex;
+      gap: 8px;
+      select {
+        flex: 1;
+      }
+      div {
+        flex: 4;
+      }
+    }
+
+    .birthday {
+      width: 100%;
+      .birthdaySelect {
+        width: 100%;
+        display: flex;
+        gap: 8px;
+        div {
+          flex: 1.4;
+          &:nth-child(2),
+          &:nth-child(3) {
+            flex: 2;
+          }
+          select {
+            width: 100%;
+          }
+        }
+      }
+    }
+
+    select {
+      height: 56px;
+      border: 1px solid $grey60;
+      border-radius: 6px;
+      padding: 17px 16px;
+      color: $grey60;
+      font-size: 16px;
+    }
+  }
+  .loginButton {
+    width: 100%;
+    padding: 36px 24px 20px 24px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    button {
+      display: flex;
+      height: 56px;
+      justify-content: center;
+      align-items: center;
+      align-self: stretch;
+      border-radius: 6px;
+      background: $blue;
+      border: 1px solid $blue;
+      cursor: pointer;
+      font-family: Apple SD Gothic Neo;
+      font-size: 18px;
+      font-weight: 800;
+      color: white;
+    }
+  }
+}
+
+.joinDone {
+  display: flex;
+  width: 100%;
+  height: 80vh;
+  padding: 0px 0px 40px;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  .backButtonContainer {
+    width: 100%;
+    display: flex;
+    button {
+      display: flex;
+      height: 56px;
+      padding: 8px 497px 8px 0px;
+      align-items: center;
+      flex-shrink: 0;
+      align-self: stretch;
+      border: none;
+      background-color: white;
+      font-size: 20px;
+      line-height: 28px;
+      cursor: pointer;
+    }
+  }
+  .joinSuccessMessage {
+    display: flex;
+    padding-bottom: 0px;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 40px;
+    flex: 1 0 0;
+    align-self: stretch;
```

유사한 속성끼리 묶어서 정렬해 주세요!
```suggestion
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    gap: 40px;
    flex: 1 0 0;
    align-self: stretch;
    padding-bottom: 0px;
```

---

리뷰하지 않은 다른 부분들도 찾아서 수정해 주세요!